warn/warn_test.go

```diff
@@ -601,6 +601,18 @@ d = {
 		[]string{"4: Dictionary items are out of their lexicographical order."},
 		scopeEverywhere)
 
+	checkFindingsAndFix(t, "unsorted-dict-items", `
+d = {
+	"deps": [],
+	"srcs": ["foo.go"],
+}`, `
+d = {
+	"srcs": ["foo.go"],
+	"deps": [],
+}`,
+		[]string{"3: Dictionary items are out of their lexicographical order."},
```

Comment:
I'm not sure if this message is clear in case priorities from `NamePriority` are used, but can't suggest anything better.